zerobuf/data/frame.fbs

```diff
@@ -0,0 +1,19 @@
+// Copyright (c) 2016, Human Brain Project
+//                     Jafet.VillafrancaDiaz@epfl.ch
+//
+namespace zerobuf.data;
+
+// Event containing the data frame range information. It is based on the same
+// units as the zerobuf.render.Frame, i.e. number of frames.
+// It may be used as a base event that defines the available range of frames
+// coming from a certain data source, being also possible to use the
+// zerobuf.render.Frame event at the same time, which would define the frame
```

Comment:
The GUI only knows what it receives from Livre, which at the moment is the render.Frame event (start, current, end, delta). Now, Livre and the GUI are using this interval of frames to loop between them in an animation, and this range can also be modified from the GUI, being possible to focus on a specific segment of the visualization.
This new data.Frame event is adding semantics for the range of frames that the data source provides, so we can easily know the available range from the GUI, but at the same time change the range within which the rendering should animate.